ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -95,3 +102,48 @@ azure_connection::get_object(const std::string &path) const
 {
     return 0;
 }
+
+int
+azure_connection::object_exists(const std::string &object_key, bool &exists) const
```

Comment:
i believe there is also an exists function too.
https://learn.microsoft.com/en-us/dotnet/api/azure.storage.blobs.specialized.blobbaseclient.exists?view=azure-dotnet#azure-storage-blobs-specialized-blobbaseclient-exists(system-threading-cancellationtoken)

Reply:
I came across that when I was researching ways to implement object_exists but that functionality is for the .NET framework.